oak-store-document/src/main/java/org/apache/jackrabbit/oak/plugins/document/Checkpoints.java

```diff
@@ -158,6 +158,17 @@ public Revision getOldestRevisionToKeep() {
         return lastAliveRevision;
     }
 
+    /**
+     * Backwards compatible method to get the oldest revision to keep
+     * with cleanup.
```

Comment:
I would suggest to keep the original comment 

```suggestion
     * Returns the oldest valid checkpoint registered.
     *
     * <p>It also performs cleanup of expired checkpoint
```

not sure it necessarily needs the comment about backwards compatible though?